migrations/versions/rev-2020-12-28-11:07:56-7374b18af581_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: 7374b18af581
+Revises: b0c55c767022
+Create Date: 2020-12-28 11:07:56.703331
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '7374b18af581'
+down_revision = 'b0c55c767022'
+
+
+def upgrade():
+	op.execute("UPDATE custom_forms SET is_public=True WHERE form='session' and (field_identifier='title' or field_identifier='short_abstract' or field_identifier='track' or field_identifier='session_type' or field_identifier='language' or field_identifier='slides' or field_identifier='video');")
```

Comment:
There is no field identifier like `short_abstract`.

It is `shortAbstract`

I was going to merge the PR, but thought to check. You should have checked the actual values before creating the PR. At this point, there should be certain credibility in the PRs you raise so that I merge them without thinking anything

Reply:
And it's not video, audio, slides

It's audioUrl, videoUrl, slidesUrl

Reply:
sorry but what are these - 
SESSION_FORM = {
    "title"
    "subtitle",
    "short_abstract",
    "long_abstract",
    "comments"
}

Reply:
These are not used. The ones in the database are fetched via:

```sql
SELECT distinct field_identifier from custom_forms where form='session';
```

Reply:
The custom form field creation was shifted to frontend and hence the above dictionary is not used anymore

Reply:
Please change the same fields in frontend PR to be public